### PR TITLE
Modifications to kinship relations

### PR DESCRIPTION
**Pull request by rjyounes**
1. Added range and property chain axioms. 
2. Added missing @en language tags.
3. Copied `adm:Tooltip` values to `skos:definitions` for kinship relations
4. Added `skos:scopeNote` annotations to clarify the meaning of more complex relationships, such as `hasAunt`, which denote more than one relationship.
5. Changed URI strings typed ^^xsd:anyURI  to resource object.

### CODE REVIEW

File: core/bdo.ttl [hunk 1]
```diff
@@ -299,8 +299,8 @@ In general include the use Copyright Page Title when it differs from the bibliog
 		;
 	adm:userTooltip "Title appearing on the copyright page of a Work, if present."@en ;
 	rdfs:seeAlso
-		"http://cip.loc.gov/cipman/"^^xsd:anyURI ,
```
Comment (eroux) [inline core/bdo.ttl:302]: that's an interesting change... the anyURI business was on purpose, with the goal to tell RDF systems not to consider these as RDF resources that might have predicates, etc.

what's your take on that?

File: core/bdo.ttl [hunk 2]
```diff
@@ -2718,13 +2751,24 @@ bdo:hasFather
 	rdfs:subPropertyOf bdo:hasParent ;
 	rdfs:label "has father"@en ;
 	adm:userTooltip "Indicates the Father of the Person"@en ;
+	rdfs:range [
+		a owl:Restriction ;
+		owl:onProperty bdo:personGender ;
+		owl:hasValue bdr:GenderMale ;
+	] ;
+	skos:definition "Indicates the Father of the Person"@en ;
```
Comment (eroux) [inline core/bdo.ttl:2759]: it's a tiny detail and I know French uses less upper case letter than English... but why an upper case F here? it makes it look very divine!

**Review by eroux — COMMENTED** [review]
thanks a lot!

File: core/bdo.ttl (hunk 1 above)
Comment (rjyounes) [reply core/bdo.ttl:302]: You're right, that was a mistake on my part. I'll change it. BTW that's a dead link, so it should probably be removed. Let me know what you want to do.

File: core/bdo.ttl (hunk 2 above)
Comment (rjyounes) [reply core/bdo.ttl:2759]: I was following the pattern used throughout BDO. If you look at the admin tooltips, which I simply copied to `skos:definition`s, you'll see that all the relationship terms are capitalized. I would also prefer to uncapitalize them. If you agree with that, I'll go through and make the changes to both annotations.

File: core/bdo.ttl (hunk 1 above)
Comment (eroux) [reply core/bdo.ttl:302]: indeed... I looked on the wayback machine and I'm not sure it was that relevant in the first place so let's remove it. Thanks!

File: core/bdo.ttl (hunk 2 above)
Comment (eroux) [reply core/bdo.ttl:2759]: ah I see yes, I think it makes more sense to use the normal (lower) case here... I think the original intent was to highlight that the words were referring to classes (sort of)... but as we encode things from the real world, we can use the case that's appropriate for these. Thanks!

File: core/bdo.ttl (hunk 1 above)
Comment (rjyounes) [reply core/bdo.ttl:302]: I asked my colleagues at Semantic Arts about the difference between the use of a typed string vs a URI resource in RDF, and the consensus is there is no reason to use the typed string in RDF generally.

There are instances where SHACL requires the use of the typed string. E.g., 

The values of `sh:namespace` are literals of datatype `xsd:anyURI`.

```
	sh:declare [
		sh:prefix "schema" ;
		sh:namespace "http://schema.org/"^^xsd:anyURI ;
	] .
```

So I've left these changes as they are, but if you have strong feelings in the other direction I can pull them out.

Comment (eroux) [reply core/bdo.ttl:302]: ah, that's interesting, thanks a lot for this clarification!

**Review by eroux — APPROVED** [review]

**Pull request merged**